src/main/scala/com/fulcrumgenomics/util/Metric.scala

```diff
@@ -176,6 +178,22 @@ trait Metric extends Product {
   /** Get the values of the arguments in the order they were defined. */
   def values: Seq[String] = productIterator.map(formatValues).toSeq
 
+  /** Gets the value of the field by name. */
+  def apply(name: String): Option[String] = {
```

Comment:
If possible I'd prefer to keep the semantics of `Map` here; i.e. `apply` returns `String` (in this case) and throws an exception for an invalid key, and `get` returns `Option[String]`.